### PR TITLE
Get S3 bucket information from configuration for templates.

### PR DESCRIPTION
**Pull request by omaciel**
When uploading ansible playbook templates to S3, get the bucket region information from configuration.

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

What is it?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `go fmt ./...` to check that my code is properly formatted
- [x] I run `go vet ./...` to check that my code is free of common Go style mistakes
- [ ] I run `make lint` to prints out coding style mistakes and fix them

### CODE REVIEW

File: pkg/services/updates.go [hunk 1]
```diff
@@ -250,7 +250,7 @@ func (s *UpdateService) WriteTemplate(templateInfo TemplateRemoteInfo, account s
 		GoTemplateRemoteName: templateInfo.RemoteName,
 		FleetInfraEnv:        envName,
 		UpdateNumber:         strconv.FormatUint(uint64(templateInfo.UpdateTransactionID), 10),
-		RepoURL:              "https://{{ s3_buckets[fleet_infra_env] | default('rh-edge-tarballs-prod') }}.s3.us-east-1.amazonaws.com/{{ update_number }}/upd/{{ update_number }}/repo",
+		RepoURL:              fmt.Sprintf("https://{{ s3_buckets[fleet_infra_env] | default('rh-edge-tarballs-prod') }}.s3.%s.amazonaws.com/{{ update_number }}/upd/{{ update_number }}/repo", *cfg.BucketRegion),
```
Comment (loadtheaccumulator) [inline pkg/services/updates.go:253]: What determines this value?

Comment (omaciel) [reply pkg/services/updates.go:253]: It's read from config.go

Comment (acosferreira) [reply pkg/services/updates.go:253]: Can we use the parameters in the same way? {{}} or %s?

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
007020a3-74e8-493c-a629-b2d21d26260e
d8bdafda-2ec1-468f-8dfd-1d0cad57bb1f

**Comment by omaciel** [comment]
@lahavyuv86  or @tomershinhar  can you take a look at why tests failed, please?

**Review by acosferreira — APPROVED** [review]
lgtm

**Pull request merged**